pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go

```diff
@@ -95,7 +95,7 @@ func (c *cloudProviderObserver) ObserveCloudProviderNames(genericListers configo
 	}
 
 	// we set cloudprovider configmap values only for some cloud providers.
-	validCloudProviders := sets.NewString("azure", "vsphere")
+	validCloudProviders := sets.NewString("azure", "gce", "vsphere")
```

Comment:
it's not `gcp` ?:)

the title suggests that it should be `gcp` and I have also found https://github.com/openshift/api/commit/21798913ff1a7f03020384e9ab3f36b052146500#diff-cee9360e003319719376a6c9189c2e48

Reply:
here the values are cloud provider in terms of k8s.. https://github.com/openshift/library-go/blob/b799e5bbbcdc88a34f4bf233bb7001cadc9d5866/pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go#L150